src/utils/queries.ts

```diff
@@ -34,7 +34,7 @@ export const constructCancelTxQuery = (txConfig: ITxConfig, newGasPrice: number)
   return queryString.stringify({
     ...unfinishedSpeedUpTxQueryParams,
     to: unfinishedSpeedUpTxQueryParams.from,
```

Comment:
Just noticed these, but can you rename these to a `cancelTxQueryParams` type var instead?

Reply:
Sure